strings/build.gradle

```diff
@@ -11,6 +14,15 @@ android {
         targetSdkVersion rootProject.androidTargetSdk
         versionCode 1
         versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles 'consumer-rules.pro'
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
```

Comment:
Yeah I made these consistent in this commit: https://github.com/getodk/collect/pull/4173/commits/879b9c6c34cb43a153b93ddb5ba8450f5e2544fa. Basically minification doesn't need to happen until you build an APK/App Bundle - that's where it decides what code it can remove from and then shrinks it. So, it doesn't make sense to have minification enabled or configured for library modules. Docs on this are pretty weak but here's an SO: https://stackoverflow.com/questions/50737180/proguard-on-multimodule-app

Reply:
Oh!